stories/dooboo-ui/Checkbox.stories.tsx

```diff
@@ -39,24 +82,19 @@ function Default(): React.ReactElement {
           boxColor={'orange'}
           labelSize={20}
           labelColor={'#000000'}
-          selectedValue={[
-            {
-              label: 'first',
-              value: 1,
-              selected: true,
-            },
-            {
-              label: 'second',
-              value: 2,
-              selected: false,
-            },
-            {
-              label: 'third',
-              value: 3,
-              selected: false,
-            },
-          ]}
+          selectedValue={defaultCheckboxData}
```

Comment:
I think `items` is better props name than `selectedValue` as a single `Checkbox`'s thing.

Reply:
Thank you for the detailed review. It's corrected. Any further review would be appreciated. 👍